README.md

```diff
@@ -1 +1,51 @@
 # project-liberation
+
+## Development setup
+
+### Preparing your environment for development
+
+1.  Install pipenv on your machine. Do not create any virtualenv, just install globally:
+       
+    Manjaro linux:  
+    ``` bash
+     sudo pacman -Syu python-pipenv
+    ```
+    Ubuntu: 
+    ``` bash
+     pip install pipenv
+    ```
+
+2.  Create new virtual environment for project with all development packages. 
+    ``` bash
+     pipenv install --dev
+    ```
+        
+    Alternatively, this can be also done using Pycharm IDE.  <br />
+    - Click "Add Interpreter" (right down corner) <br />
+    - Select Pipenv Environment <br />
+    - Choose python version, make sure that checkbox "Install packages from Pipfile" is selected <br />
+    - Confirm with OK <br />
+    New virtual environment will be created. Pycharm terminals will use this virtualenv, so there is no need to proceed step 3 any more. Skip to step 4
+
+3.  Activate newly created virtualenv: 
+    ``` bash
+     pipenv shell
+    ```
+        
+4.  Create local settings. For more details please see description in `project_liberation/settings/__init__`
+    
+5.  Install npm. In project directory, deactivate virtualenv and write:
+    ``` bash
+     npm install
+    ```
+
+### Running tests and code analysis
+
+You can run automated tests and code analysis with:
+    ``` bash
+    ./full_check.sh
+    ```
+For details please see full-check.py file, it's description is always up-to-date 
+    
+## Production setup
+    TODO
```

Comment:
Please delete chapter `Production setup`. When we will need this we will create it :)

Reply:
You mean... never? ;) I did it on purpose, if application is running in production development it should have Production setup. This way it is chance to do it. If I will remove it, this part will probably be never written.

Reply:
Done